### ISSUE TITLE
[Machine Learning] A dive in Stock Market Prediction with Step-by-Step Implementation.

### ISSUE THREAD

**Issue by borosteve** [section-engineering-education/engineering-education#4392]
### Topic Suggestion
A dive in Stock Market Prediction with Step-by-Step Implementation.
#### Proposed article introduction
Predictions and analyses of the stock market are among the most challenging tasks. In addition to the market's volatility, there are many other dependent and independent factors that affect the market's value of a certain stock as well. Because of these unknowns, no stock market expert can accurately predict when the market will rise or collapse.

As a result, the most recent advances in market research and Stock Market Prediction have started to include machine learning and its powerful algorithms.
We will discuss this in detail in this tutorial.

#### Key takeaways 
- Introduction to Stock Market Prediction
- Problem statement for Stock Market Prediction
- Program Implementation
- Creating a Training Set and a Test Set for Stock Market Prediction
- Data Processing For LSTM
- LSTM Prediction

#### Article quality
This will be a detailed article and an easy one to allow all kinds of readers to grasp the article's concept. I will write a unique article that adds value beyond other official docs and to the rest of EngEd

#### References
N/A

**Comment by ahmadmardeni1** [comment]
I have already published an article about the topic: https://www.section.io/engineering-education/stock-price-prediction-using-python/

Please feel free to suggest an alternate topic to explore. 🚀
